manifests/clamd.pp

```diff
@@ -25,7 +25,7 @@
   }
 
   service { 'clamd':
-    ensure     => running,
+    ensure     => $clamav::clamd_service_ensure,
```

Comment:
Can we be consistent here.
Either use $clamav::var everywhere and remove the params of this class
OR
Add this to the params at the top and switch it to a local scope ($clamd_service_ensure)